ui/lib/kv/addon/components/page/secret/edit.js

```diff
@@ -58,10 +58,11 @@ export default class KvSecretEdit extends Component {
       this.invalidFormAlert = invalidFormMessage;
       if (isValid) {
         const { secret } = this.args;
-        yield this.args.secret.save();
+        yield secret.save();
         this.flashMessages.success(`Successfully created new version of ${secret.path}.`);
-        // transition to parent secret route to re-query latest version
-        this.router.transitionTo('vault.cluster.secrets.backend.kv.secret');
+        this.router.transitionTo('vault.cluster.secrets.backend.kv.secret.details', secret.path, {
```

Comment:
if we do include `secret.path` on the transition we'll need to do some encoding and test with nested secret paths as well. If it was working reliably without, it's probably easiest to remove

Reply:
Yeah - I forgot we had added to try and resolve test issues. Will remove!